infra/resources/prod/function_profile.tf

```diff
@@ -0,0 +1,230 @@
+      EventsQueueName              = "events" # reference to https://github.com/pagopa/io-infra/blob/12a2f3bffa49dab481990fccc9f2a904004862ec/src/core/storage_apievents.tf#L7
+
+      BETA_USERS = data.azurerm_key_vault_secret.fn_app_beta_users.value // TODO: searching this config in the codebase show no results
+      # Enable use of templated email
+      FF_TEMPLATE_EMAIL = "ALL" // TODO: searching this config in the codebase show no results
+      # Cashback welcome message
+      IS_CASHBACK_ENABLED = "false"
+      # Only national service
+      FF_ONLY_NATIONAL_SERVICES = "true" // TODO: searching this config in the codebase show no results
+      # Limit the number of local services
+      FF_LOCAL_SERVICES_LIMIT = "0" // TODO: searching this config in the codebase show no results
+      # eucovidcert configs
+      FF_NEW_USERS_EUCOVIDCERT_ENABLED       = "true"                        // TODO: searching this config in the codebase show no results
+      EUCOVIDCERT_PROFILE_CREATED_QUEUE_NAME = "eucovidcert-profile-created" // TODO: searching this config in the codebase show no results
+
+      OPT_OUT_EMAIL_SWITCH_DATE = 1625781600
+      FF_OPT_IN_EMAIL_ENABLED   = "true" // TODO: searching this config in the codebase show no results
+
+      VISIBLE_SERVICE_BLOB_ID = "visible-services-national.json" // TODO: searching this config in the codebase show no results
+
+      # Login Email variables
+      MAGIC_LINK_SERVICE_API_KEY    = data.azurerm_key_vault_secret.ioweb_profile_function_api_key.value
+      MAGIC_LINK_SERVICE_PUBLIC_URL = format("https://%s-auth-webprof-func-01.azurewebsites.net", local.project)
+      IOWEB_ACCESS_REF              = "https://account.ioapp.it"
+      #
+
+      # UNIQUE EMAIL ENFORCEMENT
+      // TODO: searching the following 2 env config in the codebase show no results
+      FF_UNIQUE_EMAIL_ENFORCEMENT    = "ALL"
+      UNIQUE_EMAIL_ENFORCEMENT_USERS = jsonencode(split(",", data.azurerm_key_vault_secret.app_backend_UNIQUE_EMAIL_ENFORCEMENT_USER.value))
+
+      PROFILE_EMAIL_STORAGE_CONNECTION_STRING = data.azurerm_storage_account.citizen_auth_common.primary_connection_string
+      PROFILE_EMAIL_STORAGE_TABLE_NAME        = "profileEmails"
+      ON_PROFILE_UPDATE_LEASES_PREFIX         = "OnProfileUpdateLeasesPrefix-001"
+
+      MAILUP_USERNAME = data.azurerm_key_vault_secret.common_MAILUP_USERNAME.value
+      MAILUP_SECRET   = data.azurerm_key_vault_secret.common_MAILUP_SECRET.value
+      PUBLIC_API_KEY  = trimspace(data.azurerm_key_vault_secret.fn_app_PUBLIC_API_KEY.value)
+      // TODO: probably does not needed anymore due the fn which was using it is now moved in io-profile-async
+      SPID_LOGS_PUBLIC_KEY = trimspace(data.azurerm_key_vault_secret.fn_app_SPID_LOGS_PUBLIC_KEY.value)
+      AZURE_NH_ENDPOINT    = data.azurerm_key_vault_secret.fn_app_AZURE_NH_ENDPOINT.value // TODO: searching this config in the codebase show no results
```

Comment:
[this PR](https://github.com/pagopa/io-functions-app/pull/154/files#diff-8f4625a1cf4cb245a2f9e780a9a17ff6a7132b8e070997144a67499d7ff793b4L50) addressed the removal from codebase. we can safely remove this